apps/sql_data/tests/test_models.py

```diff
@@ -35,6 +37,17 @@ def test_upload_metadata(self):
             self.extract_metadata.data_source.name
             == self.sql_upload_metadata.data_source_name
         )
+        assert (
+            self.sql_upload_metadata.chunks_count
+            == self.upload_chunk.upload_metadata.chunks_count
+        )
+        assert self.sql_upload_metadata.mark_as_complete(
```

Comment:
What's happening here? Do you mind breaking this down into multiple statements to make it clear what's going on.

In fact, this test method is testing to many things. Consider breaking it down into multiple test methods.